### PR TITLE
Fix broken npos check

### PR DESCRIPTION
**Pull request by nickrandolph**
Because 4 was being added to the result of rfind, the subsequent check for npos will always fail. This also means there is a risk later on of argsStart being > argsEnd. By adjusting the logic to add 4 after the npos check, the npos check will now work and the subsequent check for argsStart >argsEnd is no longer required.

### CODE REVIEW

File: dev/AppLifecycle/AppLifecycle.cpp [hunk 1]
```diff
@@ -27,11 +29,6 @@ namespace winrt::Microsoft::ProjectReunion::implementation
             return {L"", L""};
         }
 
-        if (argsStart > argsEnd)
```
Comment (aeloros) [inline dev/AppLifecycle/AppLifecycle.cpp:30]: Add this back in.

File: dev/AppLifecycle/AppLifecycle.cpp [hunk 2]
```diff
@@ -14,11 +14,13 @@ namespace winrt::Microsoft::ProjectReunion::implementation
 {
     std::tuple<std::wstring, std::wstring> ParseCommandLine(std::wstring commandLine)
     {
-        auto argsStart = commandLine.rfind(L"----") + 4;
+        auto argsStart = commandLine.rfind(L"----");
         if (argsStart == std::wstring::npos)
```
Comment (aeloros) [inline dev/AppLifecycle/AppLifecycle.cpp:18]: Thanks!

**Review by aeloros — CHANGES_REQUESTED** [review]

**Comment by riverar** [comment]
@nickrandolph Hey Nick! Can you push a new update to close out this PR?

**Comment by aeloros** [comment]
I've moved the npos change to this branch, which will be going into main as soon as a dependent change is merged.

https://github.com/microsoft/ProjectReunion/blob/user/aeloros/startuptasks/dev/AppLifecycle/AppLifecycle.cpp

**Pull request closed without merging**